dwds/lib/src/debugging/metadata/provider.dart

```
@@ -123,4 +157,14 @@ class FileMetadataProvider implements MetadataProvider {
 
   @override
   Future<Map<String, List<String>>> get scripts => throw UnimplementedError();
+
+  @override
+  Future<void> initialize(String entrypointPath) async {
+    // The merged metadata resides next to the entrypoint.
+    var serverPath =
+        entrypointPath.replaceAll('.app.bootstrap.js', '.merged_metadata');
+    var contents = await _assetReader.metadataContents(serverPath);
+    // TODO(grouma) - parse the contents.
+    print(contents);
```

maybe throw an exception that this is unimplemented?